src/herder/HerderImpl.cpp

```diff
@@ -949,8 +949,16 @@ HerderImpl::triggerNextLedger(optional<uint32_t> ledgerSeqToTrigger,
     // We pick as next close time the current time unless it's before the last
     // close time. We don't know how much time it will take to reach consensus
     // so this is the most appropriate value to use as closeTime.
+    auto const usingManualCloseWithoutExplicitCloseTime =
```

Comment:
Nitpick: `bool` and `auto` are the same length, why not be explicit?

Reply:
That sounds fine to me, though as it turns out, later changes have eliminated that code altogether. :)